Update README.md

noname25495 opened the pull request:
This pull request resolves #XXX

- [x] I have read and agreed to the [Code of Conduct](https://github.com/movie-web/movie-web/blob/dev/.github/CODE_OF_CONDUCT.md).
- [x] I have read and complied with the [Contributing Guidelines](https://github.com/movie-web/movie-web/blob/dev/.github/CONTRIBUTING.md).
- [x] This pull request is for a README fix.
- [x] I have tested all of my changes.

noname25495 committed:
Update README.md

fanlimgames commented:
I saw that it was not looking right, so here is the fixed version of your README.

noname25495 committed:
Add new color palettes: orange, pink, and teal

Extended the color theme with three new palettes: orange, pink, and teal. Each palette includes color variations from c50 to c900 to provide a comprehensive range of shades. This enhancement allows for greater flexibility in UI design.
